presto-docs/src/main/sphinx/release/release-0.207.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.207
+=============
+
+General Changes
+---------------
+
+* Fix a planning issue for queries where correlated references were used in ``VALUES``.
+* Change server to fail on startup when configuration for file based access controllers is invalid.
+* Remove support for legacy ``JOIN ... USING`` behavior.
+* Add support for column properties.
+* Change behavior for unnesting an array of ``row`` type to produce multiple columns.
+* Add optimizer to reorder inner joins and choose the join type based on cost when statistics are available.
+* Deprecate ``reorder_joins`` session property, and replace with ``join_reordering_strategy``.
+* Deprecate ``distributed_joins`` session property and replace with ``join_distribution_type``.
+* Add ``optimizer.max-reordered-joins`` configuration property to set the maximum number of joins that
+  can be reordered at once using cost-based join reordering.
+* Add support for ``char`` type to :func:`approx_distinct`.
+* Add support for securing within-cluster communication with Kerberos authentication.
+
+Security Changes
+----------------
+
+* Remove deprecated table/view-level access control methods.
+
+Web UI Changes
+--------------
+
+* Add peak total (user + system) memory to query details UI.
+
+Hive Changes
```

Comment:
Hive Connector Changes

Reply:
We always call this 'Hive Changes'

Reply:
We started adding “Connector” to all of them for consistency. See last few releases.